wip: add CMU Delphi packages (#1)
dshemetov/Epidemiology: Epidemiology.md

  the latent counts of infections and the reproduction number for a given
  population over time, and to perform model comparison using information
  criteria.
- `r github("dajmcdon/rtestim")`: Uses a Poisson likelihood with trend filtering
  penalty to estimate the effective reproduction number (Rt) of an infectious

>   disease outbreak.

Reviewer: suggest: maybe also add a link to a publication or preprint ([this](https://www.medrxiv.org/content/10.1101/2023.12.18.23299302v1) maybe)?  @dajmcdon @jiapivialiu

  resistance (AMR) data analysis and to work with microbial and antimicrobial
  properties by using evidence-based methods and reliable reference data such
  as LPSN ([Parte *et al.* 2020](https://doi.org/10.1099/ijsem.0.004332)).
- `r github("cmu-delphi/epiprocess")`: A collection of data structures and
  methods for handling epidemiological data. Features include handling versioned
  data (point-in-time snapshots), applying functions to a sliding time window,
  support for data with multiple geographic locations, and more. The package is
  designed to be used in conjunction with the `r github("cmu-delphi/epipredict")`

>   package.

Reviewer: ```suggestion
- `r github("cmu-delphi/epiprocess")`: classes and functions for handling
  epidemiological data. Its `epi_df` class holds epidemiological time series
  data, and its `epi_archive` class holds the version history of such data. It
  facilitates common exploratory data analyses, statistical modeling, and proper
  retrospective evaluation of forecasting and other models based on the version
  history of the input data. Designed to work well with the `r github("cmu-delphi/epipredict")`
  package.
```
- wording tweaks (classes and functions; don't think epi's have heard of data structures and that's a bit of a grand description anyway, and R technicalities)
- trying to emphasize proper backtesting
- multi geos is hopefully a given
- try not to tie ourselves as much to epipredict, as there are some more general-purpose things in there as well

- `r github("cmu-delphi/epipredict")`: A framework for building epidata
  pipelines involving transformation and forecasting, built on
  `r github("cmu-delphi/epiprocess")`. It supports: a simple interface for
  defining data pre-processing, forecasting, and post-processing steps, a
  collection of data processing steps common to epidata, and a collection of

>   pre-packaged forecasting models commonly used by CMU Delphi's forecasting team.

Reviewer: ```suggestion
- `r github("cmu-delphi/epipredict")`: provides (i) a set of easy-to-use
  statistical epidemiological forecasting models, and (ii) a framework for
  creating custom forecasters from modular pre-processing, forecasting, and
  post-processing steps. Leverages `r github("cmu-delphi/epiprocess")` and `r github("cmu-delphi/epiprocess")`
  and `r pkg("tidymodels")`.
```

trying to frontload the important 2-part list

  publicly available parasite rate survey points, mosquito occurrence points and
  raster surfaces from the ['Malaria Atlas Project'](https://malariaatlas.org/)
  servers as well as utility functions for plotting the downloaded data.
- `r pkg("epidatr")`: An R client for the [Delphi Epidata
  API](https://cmu-delphi.github.io/delphi-epidata/), which provides access to a

>   wide range of epidemiological data, including COVID-19 data.

Reviewer: ```suggestion
  wide range of epidemiological data, including novel surveillance sources and
  mirrors of official reports, as well as historical versions of these data.
```

Think we could also mention epidatr under "Digital Surveillance" or is this 1 category per package?

Author: I can include this as a question when I move the PR over to their repo.
